### PR TITLE
fix: remove !important classes

### PR DESCRIPTION
**Pull request by BogdanDanilescu**
PR to remove the important classes ( `!gi...` ) with the exception of the`notyf` package styles as we need to override the native classes that come from the package.

### CODE REVIEW

**Review by ramonmulia — COMMENTED** [review]
I see some spacing changes in Chromatic, are they intentional?

**Comment by BogdanDanilescu** [comment]
> I see some spacing changes in Chromatic, are they intentional?

No it's not intentional, I'll have a look

**Comment by BogdanDanilescu** [comment]
@ramonmulia there are no more UI changes now

**Review by ramonmulia — APPROVED** [review]
LGTM

File: packages/html/ds/src/card/card.html [hunk 1]
```diff
@@ -61,7 +57,7 @@
               }}
             </div>
           {% else %}
-            <div class="{{ titleClasses }}">{{ props.title }}</div>
+            <div class="gi-card-title">{{ props.title }}</div>
```
Comment (hamza14khan) [inline packages/html/ds/src/card/card.html:60]: praise: good catch!

**Review by hamza14khan — APPROVED** [review]

**Review by pregno — APPROVED** [review]

**Pull request merged**